crates/common_enums/src/enums.rs

```diff
@@ -1177,6 +1178,28 @@ pub enum PaymentMethodIssuerCode {
     JpBacs,
 }
 
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    PartialEq,
+    Eq,
+    Hash,
+    serde::Serialize,
+    serde::Deserialize,
+    strum::Display,
+    strum::EnumString,
+    ToSchema,
+)]
+#[router_derive::diesel_enum(storage_type = "db_enum")]
+#[strum(serialize_all = "snake_case")]
+#[serde(rename_all = "snake_case")]
+pub enum PaymentMethodStatus {
+    Active,
+    Inactive,
+    Processing,
+}
+
```

Comment:
Can we derive `Default` and make `Active` status as default variant

Reply:
@Chethan-rao The default when creating a payment method entry in DB is set to `Active` for seamless backwards compatibility. It's not the actual default value for the enum since payment methods can start out as either `Active` (on-session cards) or `Processing` (3DS Cards or MIT setup with any asynchronous flows) which we will handle with checks in the upcoming MIT orchestration feature.